pmd-test/pom.xml

```diff
@@ -45,6 +45,10 @@
             <groupId>net.sourceforge.pmd</groupId>
             <artifactId>pmd-test-schema</artifactId>
         </dependency>
+        <dependency>
+            <groupId>net.sourceforge.pmd</groupId>
+            <artifactId>pmd-ant</artifactId>
+        </dependency>
         <dependency>
             <groupId>org.apache.ant</groupId>
             <artifactId>ant</artifactId>
```

Comment:
We should get rid of these at some point… just like languages don't need the CLI to test, they shouldn't need Ant. Ant is an entry point for PMD as a whole, and only pmd-dist should explicitly require it.